src/components/side_panel/chart/bar_chart/bar_chart_design_panel.xml

```diff
@@ -12,7 +12,14 @@
           />
         </div>
       </Section>
-      <ChartTitle title="title" update.bind="updateTitle"/>
+      <ChartTitle
+        title="title"
+        update.bind="updateTitle"
+        isTitleCellReferenced="isTitleCellReferenced"
+        updateIsTitleCellReferenced.bind="updateIsTitleCellReferenced"
+        titleCellReference="titleCellReference"
+        updateTitleCellReference.bind="updateTitleCellReference"
+      />
```

Comment:
That's ... a lot or props. I'm sure those are necessary, but it doesn't really fir into the concept of "Generic & easily reusable building block component".

How about just giving ChartTitle the chartId as props and letting the component handle everything ? 

Another option would be to give ChartTitle  the same props as the DesignPanel (chartDefinition/updateChart/canUpdateChart). I think I prefer the first option, so ChartTitle is very easy to use with just one props needed, but both sounds fine.

Reply:
@hokolomopo, 

With the recent merging of ANHE's major improvement task, the overall picture has changed.

The `title.ts` file is now used not only for the Chart Title but also for the Axis Title.

Should we continue to accept only one prop (figureId) and let the component handle everything, or should we accept the necessary props from the parent? 

![image](https://github.com/odoo/o-spreadsheet/assets/121081869/386b40b1-8658-499f-8272-d61763fa2cec)

Reply:
Hello :slightly_smiling_face: 

I still feel like having as few props as possible is the way to go. But now since the ChartTile is used in multiple place, we need more props than the figureId indeed. 

Since we now already have 8 props for this component, you can either add the props you need and after this task we'll try to do a follow up to refactor and reduce the number of props, or you can refactor in this task. Do as you want :)